core/templates/dev/head/profile/profile.html

```diff
@@ -31,102 +31,90 @@
-                    objective="expl.objective"
-                    category="expl.category"
-                    ratings="expl.ratings"
-                    thumbnail-icon-url="expl.thumbnail_icon_url"
-                    thumbnail-bg-color="expl.thumbnail_bg_color"
-                    num-views="expl.num_views">
-                </exploration-summary-tile>
-              </div>
-              <div ng-if="userCreatedExplorations.length === 0">
-                <em>This user hasn't created any explorations yet.</em>
-              </div>
-            </md-content>
-          </md-tab>
-          <md-tab label="Edited Explorations">
-            <md-content class="oppia-profile-tabs-content">
-              <div class="oppia-profile-explorations-container">
-                <exploration-summary-tile ng-repeat="expl in userEditedExplorations"
-                    exploration-id="expl.id"
-                    exploration-title="expl.title"
-                    last-updated-msec="expl.last_updated_msec"
-                    objective="expl.objective"
-                    category="expl.category"
-                    ratings="expl.ratings"
-                    thumbnail-icon-url="expl.thumbnail_icon_url"
-                    thumbnail-bg-color="expl.thumbnail_bg_color"
-                    num-views="expl.num_views">
-                </exploration-summary-tile>
-              </div>
-              <div ng-if="userEditedExplorations.length === 0">
-                <em>This user hasn't edited any explorations yet.</em>
-              </div>
-            </md-content>
-          </md-tab>
-        </md-tabs>
+    <md-card class="oppia-profile-content-card">
+      <strong>Portfolio</strong>
+      <md-content class="oppia-profile-portfolio-container">
+        <exploration-summary-tile ng-repeat="expl in userDisplayedExplorations()"
+          exploration-id="expl.id"
+          exploration-title="expl.title"
+          last-updated-msec="expl.last_updated_msec"
+          objective="expl.objective"
+          category="expl.category"
+          ratings="expl.ratings"
+          thumbnail-icon-url="expl.thumbnail_icon_url"
+          thumbnail-bg-color="expl.thumbnail_bg_color"
+          num-views="expl.num_views">
+        </exploration-summary-tile>
+
+        <div ng-if="userEditedExplorations.length === 0">
+          <em></br>This user hasn't created or edited any explorations yet.</em>
+        </div>
       </md-content>
+
+      <span class="oppia-profile-portfolio-pages" ng-if="numUserPortfolioExplorations > 6">
+        <button ng-disabled="currentPage == 0" ng-click="previousPage(); newExplorationNumbers()">
+        Previous
+        </button>
+        Showing <[startingExplorationNumber]> - <[endingExplorationNumber]> of <[numUserPortfolioExplorations]>
+        <button ng-disabled="currentPage >= userEditedExplorations.length/pageSize - 1" ng-click="nextPage(); newExplorationNumbers()">
```

Comment:
I think the condition is cleaner if expressed as (currentPage + 1) * pageSize >= userEditedExplorations.length